### ISSUE TITLE
packrat

### ISSUE THREAD

**Issue by asifm** [CIT-ee/VA-OZ#9]
Your full reproducibility idea was nice. But packrat is causing more problem than helping, at least in my machine. I'd suggest let's remove it. We can write `install("x")` in the code itself. You can also write down the version number in a comment next to the install command. What do you think?

**Comment by Spider101** [comment]
That is weird. I cloned the repository locally (Windows) and had no issues with packrat. For the time being, you can turn off packrat for your machine by going into your project's .Rprofile file and commenting out `source("packrat/init.R")` - it should be the only line in the file. Let me know if that resolves the issue!

**Comment by asifm** [comment]
Yes, that solved it. Let's commit that change. Do you need it for some reason?

**Comment by Spider101** [comment]
Yes, it simplifies my workflow. Let's not commit it just yet. You can keep the change locally so it works properly on your machine. I will push a requirements.txt like file later so the repo is functional w/ or w/o packrat.
